doc/docs/Materials.md

```diff
@@ -52,7 +52,7 @@ which is equivalent to the Lorentzian model except that the $\omega_n^2$ term ha
 Numerical Stability
 -------------------
 
-If a Lorentzian resonance $\omega_n$ is specified at too high a frequency relative to the time discretization $\Delta t$, the simulation becomes unstable. Essentially, the problem is that $\mathbf{P}_n$ oscillates too fast compared with the time discretization for the discretization to work properly. If this happens, there are three workarounds: increase the resolution which increases the resolution in both space and time, decrease the Courant factor which decreases $\Delta t$ compared to $\Delta x$, or use a different model function for your dielectric response.
+If a Lorentzian resonance $\omega_n$ is specified at too high frequency relative to the time discretization $\Delta t$, the simulation becomes unstable. Essentially, the problem is that $\mathbf{P}_n$ oscillates too fast compared with the time discretization for the discretization to work properly. If this happens, there are three workarounds: increase the resolution which increases the resolution in both space and time, decrease the Courant factor which decreases $\Delta t$ compared to $\Delta x$, or use a different model function for your dielectric response.
```

Comment:
The original "too high a frequency" is correct.  (Google "too high a")

Reply:
Thank you!

Reply:
Can you revert that part of the PR so that I can merge it?